Kaguya/Kaguya/Discord/Commands/Reference/Fetch.cs

```diff
@@ -0,0 +1,30 @@
+﻿using Discord.Commands;
+using Kaguya.Discord.Attributes;
+using Microsoft.Extensions.Logging;
+using System.Threading.Tasks;
+using Discord.Rest;
+using Kaguya.Discord.Attributes.Enums;
+
+namespace Kaguya.Discord.Commands.Reference
+{
+    [Module(CommandModule.Reference)]
+    [Group("fetch")]
+    public class Fetch : KaguyaBase<Fetch>
+    {
+        private readonly ILogger<Fetch> _logger;
+
+        public Fetch(ILogger<Fetch> logger) : base(logger) { _logger = logger; }
+
+        [Command]
+        [Summary("Gets some basic information about a user.")]
+        [Remarks("<user id>")]
+        public async Task FetchCommand(ulong userId = 1)
```

Comment:
Why does this have a default user id? who is user id 1?

Reply:
Whoops. Forgot to leave this on the other branch.